packages/react/src/component-tests/IcStepper/IcStepperTestData.tsx

```diff
@@ -0,0 +1,285 @@
+import React from "react";
+import { IcStepper, IcStep, IcButton } from "../../components";
+
```

Comment:
variables can be declared at top of file

```suggestion
const backBtn = document.querySelector(".back-btn-compact-stepper");
const nextBtn = document.querySelector(".next-btn-compact-stepper");
const steps = document.querySelectorAll("#standard-compact-stepper > ic-step");
const compactStepTypes = ["active", "completed", "disabled", "completed"];
const backBtnCustomCompactStepper = document.querySelector(
  ".back-btn-custom-compact-stepper"
);
const nextBtnCustomCompactStepper = document.querySelector(
  ".next-btn-custom-compact-stepper"
);
const customSteps = document.querySelectorAll(
  "#custom-compact-stepper > ic-step"
);
const customCompactStepTypes = [
  "active",
  "completed",
  "disabled",
  "completed",
  "disabled",
];
```